rcsb/utils/dictionary/DictMethodResourceCacheWorkflow.py

```diff
@@ -31,14 +31,30 @@
 
 
 class DictMethodResourceCacheWorkflow(object):
-    def __init__(self, workPath):
-        self.__cachePath = os.path.join(workPath, "workflow-output", "CACHE")
-        self.__configPath = os.path.join(workPath, "exdb-config-example.yml")
-        configName = "site_info_remote_configuration"
-        self.__configName = configName
-        self.__cfgOb = ConfigUtil(configPath=self.__configPath, defaultSectionName=configName)
-        self.__startTime = time.time()
-        logger.debug("Starting at %s", time.strftime("%Y %m %d %H:%M:%S", time.localtime()))
+    def __init__(self, **kwargs):
+        """Workflow wrapper  --  Workflow to rebuild and stash "buildable" cache resources.
+
+        Args:
+            configPath (str, optional): path to configuration file (default: exdb-config-example.yml)
+            configName (str, optional): configuration section name (default: site_info_configuration)
+            cachePath (str, optional):  path to cache directory (default: '.')
+            stashRemotePrefix (str, optional): file name prefix (channel) applied to remote stash file artifacts (default: None)
```

Comment:
Is there a reason for not including the arguments explicitly in the `def __init__()` declaration? Since these are all relevant parameters for the user, it might be valuable to include them, as well as their default values. Or if that looks messy, you could set them to `None` and then below do a `configPath = configPath if configPath else "exdb-config-example.yml`.

What are your thoughts on that? I'm fine with either that or the way you have it now, but just curious on your decision and preference.

Reply:
Ah OK, thanks for explaining. I guess for me, the main benefit to including them is that IDE's would be able to suggest the input arguments when instantiation the class. I guess the docstrings would show up too, though.

But I understand your reasoning for sake of consistency.